src/iuse_actor.cpp

```diff
@@ -4243,3 +4243,32 @@ bool deploy_tent_actor::check_intact( const tripoint &center ) const
     }
     return true;
 }
+
+void weigh_self_actor::info( const item &, std::vector<iteminfo> &dump ) const
+{
+    dump.emplace_back( "DESCRIPTION",
+                       _( "Use this item to weigh yourself.  Includes everything you are wearing." ) );
+}
+
+int weigh_self_actor::use( player &p, item &, bool, const tripoint & ) const
+{
+    // this is a weight, either in kgs or in lbs.
+    double weight = convert_weight( p.get_weight() );
+    if( weight > convert_weight( max_weight ) ) {
+        popup( "ERROR TOO HEAVY" );
```

Comment:
```suggestion
        popup( _( "ERROR TOO HEAVY" ) );
```

Reply:
Wouldn't be "ERROR: TOO HEAVY" be better than "ERROR TOO HEAVY"?

Reply:
i added the translate string and modified the message to include a colon and the max weight.